Drv/Ip/IpSocket.cpp

```diff
@@ -69,7 +69,7 @@ SocketIpStatus IpSocket::setupTimeouts(NATIVE_INT_TYPE socketFd) {
     timeout.tv_sec = this->m_timeoutSeconds;
     timeout.tv_usec = this->m_timeoutMicroseconds;
     // set socket write to timeout after 1 sec
-    if (setsockopt(socketFd, SOL_SOCKET, SO_SNDTIMEO, (char*)&timeout, sizeof(timeout)) < 0) {
```

Comment:
Isn't this cast necessary for VxWorks?

Reply:
Arrrggggg..... you're right, why does VxWorks have to be so quirky. On everything else option_value value is `void *`, but no, VxWorks decides that this is the perfect opportunity to throw in a random `char *`.

Reply:
I missed a critical detail....this code is not used on vxworks, as it is the timeout code.  We can revert to what you had before.